File: python/28b_1367A_ShortSubstrings.py
def main():
    ans = []
    t = int(input())
    for _ in range(t):
        b = input().strip()
        j, temp = 0, ""
        while j < len(b):
            temp += b[j]
            j += 2
        j -= 1
        temp += b[j]
        ans.append(temp)
    for i in ans:
        print(i)

if __name__ == "__main__":
    main()

File: cpp/28b_1367A_ShortSubstrings.cpp
#include<bits/stdc++.h>

int main(){
    int t, j;
    std::string b, temp;
    std::vector<std::string> ans;
    std::cin>>t;
    for(int i = 0; i < t; i++){
        std::cin>>b;
        temp = "", j = 0;
        while (j < b.size()){
            temp += b[j];
            j += 2;
        }
        j--;
        temp += b[j];
        ans.push_back(temp);
    }
    for(auto i : ans){
        std::cout<<i<<"\n";
    }
    return 0;
}